Rakefile

```diff
@@ -12,7 +12,7 @@ RUBY_REPO_VERSION = File.read('./.ruby-version').rstrip
 XCODE_WORKSPACE = 'WordPress.xcworkspace'
 XCODE_SCHEME = 'WordPress'
 XCODE_CONFIGURATION = 'Debug'
-EXPECTED_XCODE_VERSION = File.read('.xcversion').rstrip
+EXPECTED_XCODE_VERSION = File.read('.xcode-version').rstrip
```

Comment:
`.xcode-version` is the [recommended name](https://docs.fastlane.tools/actions/xcversion/) for the file